src/drivers/virtio/virtqueue/packed.rs

```diff
@@ -863,26 +863,26 @@ impl EventSuppr {
 }
 
 impl DrvNotif {
-	/// Enables notifications by setting the LSB.
+	/// Enables notifications by unsetting the LSB.
 	/// See Virito specification v1.1. - 2.7.10
 	fn enable_notif(&mut self) {
 		self.raw.flags = 0;
 	}
 
-	/// Disables notifications by unsetting the LSB.
+	/// Disables notifications by setting the LSB.
 	/// See Virtio specification v1.1. - 2.7.10
 	fn disable_notif(&mut self) {
-		self.raw.flags = 0;
+		self.raw.flags = 1;
 	}
 
 	/// Enables a notification by the device for a specific descriptor.
 	fn enable_specific(&mut self, at_offset: u16, at_wrap: u8) {
 		// Check if VIRTIO_F_RING_EVENT_IDX has been negotiated
 		if self.f_notif_idx {
-			self.raw.flags |= 1 << 1;
+			self.raw.flags = 2;
 			// Reset event fields
 			self.raw.event = 0;
```

Comment:
Since we specifically unset the wrap bit now, do we still need this line?

Reply:
True. It was the same case before, but I will fix it. 👍